client/src/pages/ExpandedModule.jsx

```diff
@@ -157,7 +162,7 @@ function ExpandedModule({ profile }) {
             <Link to="/expanded-module" state={{ id: parent }} className={styles.backButton}>
               Back
             </Link>
-            <Module title={title} body={body} attachments={attachments} child={children} link={moduleImage} />
+            <Module title={title} body={body} attachments={attachments} child={children} link={moduleImage} role={currRole} deleteChild={deleteChild} />
             {console.log(moduleImage)}
           </div>
           {ExpandedModuleForm}
```

Comment:
something I just noticed but u can merge all four of these lines with just {parent != null ? <Link to="/expanded-module" state={{ id: parent }} className={styles.backButton}> : <Link to="/modules">}, similar for admin, {currRole == 'admin' && ExpandedModuleForm}